docs/starters.yml

```diff
@@ -1,3 +1,22 @@
+- url: https://gatsby-wordpress-libre.netlify.com/
+  repo: https://github.com/armada-inc/gatsby-attila-theme-starter
```

Comment:
It looks like this repo points to a different repository than one that would presumably be named `gatsby-wordpress-libre-starter`?

Is this related to #22175?

Reply:
I have made changes. it was an error from my side 👍 and it is not related to #22175